openapi/src/test/java/io/helidon/openapi/StaticFileOnlyOpenApiFeatureImpl.java

```diff
@@ -15,15 +15,13 @@
  */
 package io.helidon.openapi;
 
-import java.io.StringReader;
 import java.util.Date;
 import java.util.HashMap;
 import java.util.Map;
 
 import jakarta.json.Json;
 import jakarta.json.JsonBuilderFactory;
 import jakarta.json.JsonObject;
-import jakarta.json.stream.JsonParser;
 import org.yaml.snakeyaml.Yaml;
 
 public class StaticFileOnlyOpenApiFeatureImpl extends OpenApiFeature {
```

Comment:
This should not be public, nor should the builder method be public - it is an `Impl` class

Reply:
Removed `public` from the class.